calculator.js

```diff
@@ -1,78 +1,77 @@
-
-
 // Append number or operator to the display
 function appendToDisplay(value) {
-    let display = document.getElementById('display');
+	let display = document.getElementById("display");
 
-    display.value += value;
+	display.value += value;
 }
 
 // Clear the entire display
 function clearDisplay() {
-    let display = document.getElementById('display');
+	let display = document.getElementById("display");
 
-    display.value = '';
+	display.value = "";
 }
 
 // Delete the last character from the display
 function deleteLast() {
-    let display = document.getElementById('display');
+	let display = document.getElementById("display");
 
-    display.value = display.value.slice(0, -1);
+	display.value = display.value.slice(0, -1);
 }
 
 // Calculate and show the result
 function calculateResult() {
-    let display = document.getElementById('display');
+	let display = document.getElementById("display");
 
-    try {
-        display.value = eval(display.value);
-    } catch (error) {
-        display.value = 'Error';
-    }
+	try {
+		display.value = eval(display.value);
+	} catch (error) {
+		display.value = "Error";
+	}
 }
 
 // TODO: Fix the square root function not displaying
-function calculateSquareRoot(value=null) {
-    let v;
-    if (!value) {
-        let display = document.getElementById('display');
-        v = display.value;
-    } else {
-        v = value;
-    }
-    return Math.sqrt(v);
+function calculateSquareRoot(value = null) {
+	let v;
+	if (!value) {
+		let display = document.getElementById("display");
+		v = display.value;
+	} else {
+		v = value;
+	}
+	return Math.sqrt(v);
 }
 
 // TODO: Implement square function
 function calculateSquare() {
-    // Hint: Use exponentiation
-    let display = document.getElementById('display');
-    display.value = display.value ** 2;
+	// Hint: Use exponentiation
+	let display = document.getElementById("display");
+	display.value = display.value ** 2;
 }
 
 // BUG: FIX reciprocal function
 function calculateReciprocal() {
-    // Hint: Reciprocal is 1/x
-    display.value = 1-value;
+	// Hint: Reciprocal is 1/x
+	let display = document.getElementById("display");
+	display.value = 1 / value;
```

Comment:
Value doesn't work because it isn't defined, you probably wanted it to be display.value instead.

Reply:
thank you!